---
PhoXiSize[] supported_capturing_modes
string message
bool success